Cloud/Docker/DuktapeBuilder/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM swift:latest
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV SHELL=bash
+
+RUN apt-get -y update && apt-get -y upgrade
+RUN apt-get install -y git make clang bc python-pip
+
+# Dependency of the duktape build process
+RUN pip install pyyaml
+
+RUN useradd -m builder
+
+# RUN git clone https://github.com/svaarala/duktape /home/builder/duktape
+RUN git clone https://github.com/svaarala/duktape.git /home/builder/duktape
+WORKDIR /home/builder/duktape
+
+# Docker will attempt to cache the output of every step. That's fine (and useful to speed things up, e.g. by avoiding
+# the need to download the entire source repository again every time!). However, whenever the following ARG is changed
+# (i.e. we are building a new version of the engine), a cache miss occurs (because the build context changed) and all
+# steps from here on are rerun. That, however, means we might be operating on an old checkout of the source code from
+# the cache, and so we update it again before checking out the requested revision.
```

Comment:
I think this part is missing? Since you are on the master branch after the git clone, I think you can just do `git pull && git checkout $rev`. I think the other scripts (e.g. https://github.com/googleprojectzero/fuzzilli/blob/master/Cloud/Docker/JSCBuilder/Dockerfile#L25n) are currently not doing this correctly (they should `git pull`, not `git fetch`), but there it doesn't really matter since they'll check out a specific commit, not master. I'll go fix it anyway

Reply:
Fixed this now for the other Dockerfiles: https://github.com/googleprojectzero/fuzzilli/commit/5aa6f4ef3724f197a0fc55ffc9b8caf8c809ad56